assets/js/app.js

```diff
@@ -21,15 +21,7 @@ import 'jquery.scrollto';
 import 'jquery-ui/ui/widget';
 import 'jquery-ui/ui/widgets/sortable';
 import 'bootstrap';
-import moment from 'moment';
```

Comment:
We have some code related to moment https://github.com/sonata-project/SonataAdminBundle/blob/18732411a37bd7d8831dcafcbb3f8fd462e8dd7a/src/Twig/CanonicalizeRuntime.php#L22

What should we do about it ?

Reply:
The ideal would be to move it to form extensions too, and deprecate it here. When upgrading the date picker library it wont be a dependency anymore.

We can move the function to form extensions and:

1. Use here the new function
2. On next major of form extensions the function will always return null (and will be deprecated to remove on 3.0)
3. Here we have a logic to not do anything if it returns null. When compat for form-extensions 1.x ends we remove the calls.